### PR TITLE
Added post on robjhyndman.com

### PR DESCRIPTION
**Pull request by robjhyndman**

### CODE REVIEW

File: content/post/2017-05-14-robjhyndman-site.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+title: Converting robjhyndman to blogdown
+---
+
+## Some history
+
+I started my website in 1993, only a few months after [the WWW was put into the public domain](https://home.cern/topics/birth-web). I began by looking at some existing sites to see how html worked, and then writing my own handcrafted html files. I maintained those html files for about 13 years, through several domain name changes as the web matured, and as I moved between institutions.
+
+In 2006 I was convinced that using a content management system would make my life easier, and I converted the site to Joomla. A couple of years later, Wordpress seemed to have more features, so I converted from Joomla to Wordpress where it stayed for about 9 years. Here is an image of what it looked like in 2009 just after converting to Wordpress.
+
+![robjhyndman.com in 2009 on Wordpress](/images/rjh2009.png)
```
Comment (yihui) [inline content/post/2017-05-14-robjhyndman-site.md]: Could you replace the image URL with https://cloud.githubusercontent.com/assets/163582/26041812/8ba5a59e-38f5-11e7-9698-9696d5c08539.png and delete the image under `static/`? The file size is nearly 1Mb and we hope to keep the repo lightweight.

File: content/post/2017-05-14-robjhyndman-site.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+date: 2017-05-15
+title: Converting robjhyndman to blogdown
```
Comment (yihui) [inline content/post/2017-05-14-robjhyndman-site.md]: Could you add at least the two fields `author` and `description`? An example: https://github.com/rbind/support/blame/master/content/post/2017-04-25-an-introduction-to-yihui-website.md

**Comment by yihui** [comment]
@thercast Just a quick note: when you merge the PR, please add `close #13` in the commit message. Thanks!

**Comment by yihui** [comment]
The image file under `static/` was not deleted as requested, but that is no big deal. I just deleted it. Thanks!

**Comment by robjhyndman** [comment]
Sorry

**Pull request merged**